queue/queue.php

```diff
@@ -124,6 +124,12 @@ class="dcf-w-100% dcf-input-text"
 			<div ng-if="selectedAppointment.phoneNumber != null" ng-cloak>
 				<span><b>Phone Number:</b> {{selectedAppointment.phoneNumber}}</span>
 			</div>
+			<div ng-cloak>
+				<span><b>Visa Type:</b> {{(selectedAppointment.visa || "None").trim()}}</span>
+			</div>
+			<div ng-if="selectedAppointment.appointmentType != null" ng-cloak>
```

Comment:
FWIW I don't think `appointmentType` would every be `null`

Reply:
Yup, removed the ng-if